documentation/modules/ROOT/pages/configuration/topic-auto-create-config.adoc

```diff
@@ -13,19 +13,29 @@
 
 toc::[]
 
-{prodname} automatically creates *internal* topics for offsets, connector status, config
-storage and history topics. The destination topics for the captured tables will be
-automatically created with a default config by the Kafka brokers when
-`auto.create.topics.enable` is set to `true`.{empty} +
-When topic creation is disabled on the brokers, for example in production environments,
-or when the topics need a different configuration then these topics have to be created
-upfront either automated in a custom deployment process or manually until Kafka Connect 2.6.
+//{prodname} automatically creates a variety of *internal* topics to store data for tables that track offsets, connector status, configuation
+//storage, and history.
+Depending on your Kafka version, there are two different methods for enabling automatic topic creation.
+In recent versions of Kafka, when automatic topic creation is enabled, you can specify unique topic configurations for different sets of topics
 
-Since Kafka 2.6.0 Kafka Connect supports customizable topic auto-creation.
+In Kafka versions earlier than 2.6.0, you can enable automatic topic creation by setting the `auto.create.topics.enable` property on the Kafka broker to `true`,
+In this instance, the Kafka broker automatically creates destination topics for the tables that it captures.
+//applies to internal and external topics?
+However, the resulting topics all share a single default configuration.
+You do not have the option to specify unique configurations for different set of topics.
+If you want to create a topic with a specific configuration, you must disable automatic topic creation, and then create the topic ahead of time, either manually, or through a custom deployment process.
 
+
+// Type: procedure
+// ModuleID: configuring-automatic-topic-creation-in-kafka-2-6
+// Title: Configuring automatic topic creation in Kafka 2.6 and later
+[id="enabling-automatic-topic-creation"]
 == Set up Kafka Connect
 
-Kafka Connect since Kafka 2.6.0 comes with topic creation enabled:
+In Kafka 2.6.0, the introduction of a new property provides greater flexibility in configuring automatic topic creation.
+You set the `topic.creation.enable` property in Kafka Connect to enable or disable automatic topic creation.
+When automatic topic creation is enabled by this property, you can define multiple configuration profiles and apply them to groups of topics based on topic names.
+The default value for the property is `true`, enabling automatic topic creation, as shown in the following example:
```

Comment:
So I think this is confusing, as there's two different concepts at play here:

* auto-topic creation by the broker; this has existed before, and means the broker will automatically create a missing topic when receiving a message for a non-existent topic; this is generally not recommended to use as you'll end up with only a single setting for all those auto-created topics
* topic creation via Kafka Connect: this is new, and here Kafka Connect does create any missing topics, based on the connector configuration; i.e. topics are _not_ auto-created by the broker

The two concepts should not be confused in the docs.

Reply:
Yes, I'll do a better job of clarifying the distinction.